test/OrchardCore.Tests/Shell/ShellHostTests.cs

```diff
@@ -38,13 +34,12 @@ public async Task FindCaseInsensitiveShellSettings(string shellName)
     }
```

Comment:
```csharp
Assert.NotEqual(shellName, anotherVariantOfShellName);
```

This will never fail while the values are different. Please let us know which values should be tested?

Reply:
It will fail if the user accident provided equal parameters in the InlineData. I would leave it just in case someone made a change to the test. If both parameters are equal the test will pass but won't really test anything.

Reply:
Cool, you are testing the test ;)

Reply:
Yup